robottelo/host_helpers/cli_factory.py

```diff
@@ -726,10 +726,14 @@ def _setup_org_for_a_rh_repo(self, options=None):
         # If manifest does not exist, clone and upload it
         if len(self._satellite.cli.Subscription.exists({'organization-id': org_id})) == 0:
             with clone() as manifest:
-                self._satellite.put(manifest.path, manifest.name)
+                remote_name = gen_alpha()
```

Comment:
why not use upload_manifest helper here instead?
```
   with clone() as manifest:
        self._satellite.upload_manifest(org.id, manifest.content)
```

Reply:
Of course, that's much better. Thanks!